nengo_bones/templates/base_script.sh.template

```diff
@@ -25,12 +25,16 @@ fi
 
 if [[ "$COMMAND" == "install" ]]; then
     {% block install %}
+    {% if conda_install %}
+    exe conda install {%- for pkg in conda_install %} "{{ pkg }}" {%- endfor %}
```

Comment:
I don't think `conda` isn't available to most uses of the `base_script` template. NengoBones is opinionated in some places, and using `pip` is one of them. If you want to use conda you'll have to change more things, and because of that I would force the downstream repo to extend one of the templates and override the `install` block to do `conda` stuff.

Reply:
Oh.. Okay. I'll remove the conda stuff then.

Reply:
What about putting it just in the `remote.sh.template` file? Right before here: https://github.com/nengo/nengo-bones/blob/master/nengo_bones/templates/remote.sh.template#L115 ?

My reason for putting this in is specifically for pandoc. With the other CI scripts, we have the `apt_install` option. But, for remote scripts, doing that is not possible because permissions aren't available. I.e., the only way to have pandoc available is either: get the system administrator to install it, or use conda.

Reply:
You could just put
```
pre_commands:
    - conda install pandoc
```
in the configuration of your docs script and that would achieve the same goal I believe.